docs/api/covidcast-signals/nhsn.md

```diff
@@ -0,0 +1,91 @@
+---
+title: NHSN ED Visits
+parent: Data Sources and Signals
+grand_parent: Main Endpoint (COVIDcast)
+nav_order: 1
+---
+# National Syndromic Surveillance Program Emergency Department Visits
+{: .no_toc}
+
+* **Source name:** `nhsn`
+* **Earliest issue available:** August 08, 2020
+* **Number of data revisions since 18 Nov 2024:** 0
+* **Date of last change:** Never
+* **Available for:** state, hhs, nation (see [geography coding docs](../covidcast_geography.md))
+* **Time type:** week (see [date format docs](../covidcast_times.md))
+* **License:** [Public Domain US Government](https://www.usa.gov/government-works)
+
+## Overview
+
+[The National Healthcare Safety Network (NHSN)](https://www.cdc.gov/nhsn/index.html) is the nation’s most widely used healthcare-associated infection tracking system.
+This dataset represents preliminary weekly hospital respiratory data and metrics aggregated to national and state/territory levels reported to CDC’s National Health Safety Network (NHSN) beginning August 2020.
```

Comment:
todo: clarify wording "beginning August 2020". Unclear if this is referring to issue date or reference date.